pkgs/tools/misc/trezorCipherKeyValue/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "trezorCipherKeyValue-${version}";
+  version = "009cfa43c8105bc56e20ce123f8def68b480edd8";
```

Comment:
nixpkgs prefers tagged releases over git hashes for versions

Reply:
I can tag only my forked versions, not origin ones. That will stay on commit version

Reply:
If upstream doesn't have version numbers the convention is to use the commit date as the package version, so no fork needed.